tools/custom-action/Dockerfile

```diff
@@ -54,10 +55,10 @@ COPY wgsl /wgsl
 RUN \
   source /dependency-versions.sh && \
   cd /wgsl && \
-  make grammar/grammar.js SHELL=/bin/bash && \
+  script -q -c "make grammar/grammar.js SHELL=/bin/bash" /dev/null && \
```

Comment:
I don't understand why we're building and installing the WGSL parser in the docker image.  The parser is dependent on the spec sources, and will vary from one PR to another.   For the longer term it makes sense to me to do this work in the entry point script instead of in the docker image. 
But this PR does not make it worse, so I approve it.

Reply:
@dneto0 it is actually intentional, this acts like a cache (copied to the working directory), since dependency versions stay stable for a long time; however, fetching them fresh every time sometimes leads to network problems, etc., with npm and alike, and current approach made things better at the cost of a slight increase in image disk size. Thank you for the review! After image builds, I will run the failing CI actions then those should pass.